docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+- A Digital Asset Management (DAM) help to crawl in your image ressources to use and reuse them in your content. And a DAM connector allows to also search in your images hosted on third party DAM servers.
+
+#### [[= product_name_headless =]] integrates into the networks
+
+##### Intranets and extranets
+
+- [[= product_name_connect =]] is to create application interconnections with low code, drag-and-drop and a very visual interface.
+  Complex data flows can be easily implemented with a huge library of connectors and actions for famous to specific applications.
+  See [[[= product_name_connect =]] product guide]([[= connect_doc =]]/general/ibexa_connect/) for more.
+- An OAuth 2 server offers the possibility to use the DXP as the authentification service for other applications.
+- An OAuth 2 client supports authentication with a third-party OAuth 2 server.
+- A DAM Connector, previously mentioned, helps to access any image repository when needing to illustrate a content.
+- [[= product_name_headless =]] supports Elasticsearch and Solr.
+  It gives the choice between using Solr or Elasticsearch as a search engine, whether hosted on [[= product_name_cloud =]] or on-premises.
+  This choice might be influenced by technology you already use, or you want to invest in for other internal projects.
+- [[= product_name_headless =]] offers to export and import from command line part of the content model or content items.
+  For example, it can be used to move new content types and items from a staging instance to the production one.
+
+##### Internet, delivery, web search engines, and social networks
+
+- [[= product_name_headless =]] comes with the support of Fastly content delivery network (CDN).
+  The HTTP cache varies on current user's role and is purged when content change.
+  With is huge network of points of presence (POP) around the world, Fastly is quickly delivering cached content from nearest server for a better user experience.
+- A Search Engine Optimization (SEO) field implements best practices about web search engine indexing and social network sharing.
+  It covers canonical URLs which are mandatory if multiple locations are used for a same Content item to avoid duplicate content,
+  Open Graph protocol to better describe a Content item to social networks and search engine, and Twitter Cards.
+
+## Use cases
+
+- A website to inform about your organization
+- An online catalog to expose your products and convince potential customer to contact you
+- A collaborative workspace to produce content, and feeds providing content to partners' or parent organization's network
+- A feed to a phone app allowing to consult instruction manuals with mobility while using your products
+- A structuring first step to create online presence, before increasing user experience with [[= product_name_experience =]], and finally becoming an online store with [[= product_name_commerce =]].
```

Comment:
Product name placeholders did not convert properly

Reply:
```suggestion
- A structuring first step to create online presence, before increasing user experience with [[= product_name_exp =]], and finally becoming an online store with [[= product_name_com =]].
```